src/main/webapp/app/components/oncokbTable/OncoKBTable.tsx

```diff
@@ -50,17 +51,34 @@ export default class OncoKBTable<T> extends React.Component<
   render() {
     return (
       <div>
-        {this.props.disableSearch ? undefined : (
-          <div className="d-flex">
-            <div className="ml-auto">
-              <input
-                onChange={(event: any) => {
-                  this.searchKeyword = event.target.value.toLowerCase();
-                }}
-                className="form-control input-sm"
-                type="text"
-                placeholder="Search ..."
-              />
+        {this.props.filters === undefined && this.props.disableSearch ? (
```

Comment:
What's the purpose of adding this additional check? Prevent rendering extra div?

Reply:
yes, the mt-2 row div creates 8-pixel wide blank row due to the margin. @zhx828